src/Symfony/Bridge/PhpUnit/DeprecationErrorHandler.php

```diff
@@ -98,15 +98,14 @@ public static function register($mode = 0)
             'legacyCount' => 0,
             'otherCount' => 0,
             'remaining vendorCount' => 0,
-            'unsilenced' => array(),
-            'remaining' => array(),
-            'legacy' => array(),
-            'other' => array(),
-            'remaining vendor' => array(),
+            'unsilenced' => [],
+            'remaining' => [],
+            'legacy' => [],
+            'other' => [],
+            'remaining vendor' => [],
         ];
         $deprecationHandler = function ($type, $msg, $file, $line, $context = array()) use (&$deprecations, $getMode, $UtilPrefix, $inVendors) {
-            $mode = $getMode();
-            if ((E_USER_DEPRECATED !== $type && E_DEPRECATED !== $type) || DeprecationErrorHandler::MODE_DISABLED === $mode) {
+            if ((E_USER_DEPRECATED !== $type && E_DEPRECATED !== $type) || self::MODE_DISABLED === $mode = $getMode()) {
```

Comment:
self:: should be reverted for the same reason

Reply:
ok

Reply:
This PR is closed -> https://github.com/symfony/symfony/pull/30490